src/picocon/index.php

```diff
@@ -369,7 +369,7 @@
 				If you&apos;re curious, we have a record of <a href="<!--SRVROOT-->/history/">Picocons of years gone by</a>.
 			</p>
 			<p class="copyright">Imperial College Science Fiction Society. Please report issues to
-			<a href="mailto:icsflibrary@gmail.com">icsflibrary@gmail.com</a></p>
+			<a href="mailto:icsflibrary@gmail.com">icsf.techpriest@gmail.com</a></p>
```

Comment:
You didn't change the clickable part ;)

Reply:
Oh, I'm so dumb! Thanks for pointing it out.

Reply:
Not dumb; change blindness is a very real thing in code :3